snuba/settings/settings_test_distributed_migrations.py

```diff
@@ -0,0 +1,64 @@
+import os
+from typing import Any, Mapping, Sequence
+
+CLUSTERS: Sequence[Mapping[str, Any]] = [
+    # {
+    #     "host": os.environ.get("CLICKHOUSE_HOST", "clickhouse"),
+    #     "port": int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+    #     "user": os.environ.get("CLICKHOUSE_USER", "default"),
+    #     "password": os.environ.get("CLICKHOUSE_PASSWORD", ""),
+    #     "database": os.environ.get("CLICKHOUSE_DATABASE", "snuba_test"),
+    #     "http_port": int(os.environ.get("CLICKHOUSE_HTTP_PORT", 8229)),
+    #     "storage_sets": {},
+    #     "single_node": False,
+    #     "cluster_name": "query_cluster",
+    #     "distributed_cluster_name": "query_cluster",
+    # },
+    {
+        "host": os.environ.get("CLICKHOUSE_HOST", "clickhouse-01"),
+        "port": int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+        "user": os.environ.get("CLICKHOUSE_USER", "default"),
+        "password": os.environ.get("CLICKHOUSE_PASSWORD", ""),
+        "database": os.environ.get("CLICKHOUSE_DATABASE", "snuba_test"),
+        "http_port": int(os.environ.get("CLICKHOUSE_HTTP_PORT", 8229)),
+        "storage_sets": {
+            "migrations",
+        },
+        "single_node": False,
```

Comment:
so I actually think we could make the migrations cluster a single node, sine that's what it is production as of right now

Reply:
Ok I'll try that out. My understanding was setting single_node would disable the dist tables.